src/lobster/model/modern_bert/_modern_bert.py

```diff
@@ -9,7 +10,11 @@
 from ._config import FlexBertConfig
 from ._model import FlexBertModel, FlexBertPredictionHead
 
-from flash_attn.losses.cross_entropy import CrossEntropyLoss
+_FLASH_ATTN_AVAILABLE = False
+
+if find_spec("flash_attn"):
+    from flash_attn.losses.cross_entropy import CrossEntropyLoss
```

Comment:
Can we just use regular torch CrossEntropyLoss if flash attention isn't available?